core/src/main/scala/org/apache/spark/deploy/master/ApplicationInfo.scala

```diff
@@ -75,9 +75,9 @@ private[deploy] class ApplicationInfo(
     }
   }
 
-  private[master] def addExecutor(worker: WorkerInfo, cores: Int, useID: Option[Int] = None): 
-  ExecutorDesc = {
-    val exec = new ExecutorDesc(newExecutorId(useID), this, worker, cores, desc.memoryPerSlave)
+  private[master] def addExecutor(worker: WorkerInfo, cores: Int, useID: Option[Int] = None):
+      ExecutorDesc = {
```

Comment:
I appreciate the attempt to fix this, but the style is still wrong. It should be:
```
private[master] def addExecutor(
    worker: WorkerInfo,
    ...): ExecutorDesc = {
  ...
}
```